@xen-orchestra/lite/src/stories/web-core/checkbox/ui-checkbox.story.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+  <ComponentStory
+    v-slot="{ properties, settings }"
+    :params="[
+      prop('accent').required().enum('brand', 'success', 'warning', 'danger').preset('brand').widget(),
+      prop('disabled').bool().widget(),
+      slot(),
+      slot('info'),
+      setting('defaultSlot').widget(text()).preset('Label'),
+      setting('info').widget(text()).preset('Lorem ipsum dolor sit amet, consectetur adipiscing elit.'),
```

Comment:
To document the `modelValue` handling on the input:
```suggestion
      setting('info').widget(text()).preset('Lorem ipsum dolor sit amet, consectetur adipiscing elit.'),
      model().type('boolean').preset(false)
```